docs/en/04_Changelogs/4.0.0.md

```diff
@@ -843,29 +574,29 @@ If upgrading from an existing installation, make sure to invoke `?flush=all` at
 
 See [/developer_guides/files/file_security] for more information.
 
-### `ListboxField` is now multiple-only
+#### `ListboxField` is now multiple-only
 
 Previously, this field would operate as either a single select (default) or multi-select by setting
 `setMultiple` to either true or false.
 
 Now this field should only be used for multi-selection. Single-selection should be done using
 a regular `DropdownField`.
 
-### `GroupedDropdownField::setDisabled` now only accepts a list of values.
+#### `GroupedDropdownField::setDisabled` now only accepts a list of values.
 
 Where previously you could specify a list of grouped values in the same way as `setSource`, this
 method now only accepts either a non-associative array of values (not titles) or an `SS_List`
 of items to disable.
 
-### Upgrading Requirements file paths
+#### Upgrading Requirements file paths
```

Comment:
Should be grouped with the section about "Update static references to asset paths"